assets/js/util/when-active.js

```diff
@@ -39,30 +39,37 @@ const { useSelect } = Data;
  *
  * @since 1.16.0
  *
- * @param {Object}   options                     Options for enhancing function.
- * @param {string}   options.moduleName          Name of a module to check.
- * @param {Function} [options.fallbackComponent] Optional. Fallback component to render when the module is not active.
+ * @param {Object}                options                       Options for enhancing function.
+ * @param {string}                options.moduleName            Name of a module to check.
+ * @param {WPComponent|undefined} [options.FallbackComponent]   Optional. Fallback component to render when the module is not active.
+ * @param {WPComponent|undefined} [options.IncompleteComponent] Optional. Fallback component to render when the module is active but not connected.
```

Comment:
It should be `{WPComponent|null}`:

```suggestion
 * @param {WPComponent|null} [options.FallbackComponent]   Optional. Fallback component to render when the module is not active.
 * @param {WPComponent|null} [options.IncompleteComponent] Optional. Fallback component to render when the module is active but not connected.
```